README.md

```diff
@@ -7,7 +7,12 @@ Metering records historical cluster usage, and can generate usage reports showin
 
 ## Project status: alpha
 
-Most major planned features are implemented, however the API is still subject to change.
+Read more about the implemented and planned features in the documentation:
```

Comment:
These could all probably come after the installation section rather than preceding it.

Reply:
I thought it was more important to highlight what it is and what it will become before install given the alpha nature

Reply:
how about we move install into this list?

Reply:
I just want it to be obvious which section you start with, and it should probably be installation since it is the entrypoint into other docs. This means putting install before other links, and making sure it's fairly obvious. We can move install into the list, but it should be the first link, and we should probably call it out as what most people should start with.

Reply:
Updated